lib/sprig.rb

```diff
@@ -19,6 +19,12 @@ module Sprig
   autoload :Seed,                 'sprig/seed'
 
   class << self
+    attr_writer :adapter
+
+    def adapter
+      @adapter ||= :active_record
```

Comment:
I always use class ivars instead of class vars, since class variables behave counter-intuitively around inheritance. Since this is your sandbox, though, I'll go ahead and change to a class var.